### PR TITLE
AVRO-3329 Added omitted braces to CodeGen class

### PR DESCRIPTION
**Pull request by KyleSchoonover**
Additionally addressed SA1131 - moved literal value of comparison to the right side.

### Jira

- [ ] My PR addresses the following [Avro Jira](https://issues.apache.org/jira/browse/AVRO/) issues and references them in the PR title. For example, "AVRO-1234: My Avro PR"
  - https://issues.apache.org/jira/browse/AVRO-3329

### Tests

- [ ] My PR does not need testing for this extremely good reason: Non functional changes

### Commits

- [ ] My commits all reference Jira issues in their subject lines. In addition, my commits follow the guidelines from "[How to write a good git commit message](https://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
  - All the public functions and the classes in the PR contain Javadoc that explain what it does

### CODE REVIEW

File: lang/csharp/src/apache/main/CodeGen/CodeGen.cs [hunk 1]
```diff
@@ -220,7 +227,10 @@ protected virtual SchemaNames generateNames(Schema schema)
         protected virtual void addName(Schema schema, SchemaNames names)
         {
             NamedSchema ns = schema as NamedSchema;
-            if (null != ns) if (names.Contains(ns.SchemaName)) return;
+            if (ns != null && names?.Contains(ns.SchemaName) == true)
```
Comment (martin-g) [inline lang/csharp/src/apache/main/CodeGen/CodeGen.cs:230]: Why names`?` ?
AFAIS it cannot be `null`.
Now GitHub reports the following for line 250:
```
Check warning on line 250 in lang/csharp/src/apache/main/CodeGen/CodeGen.cs

Code scanning

Dereferenced variable may be null

Warning
Variable
may be null here as suggested by
null check.
Show more details 
```

Comment (martin-g) [reply lang/csharp/src/apache/main/CodeGen/CodeGen.cs:230]: Any reason to add `== true` ?

Comment (KyleSchoonover) [reply lang/csharp/src/apache/main/CodeGen/CodeGen.cs:230]: Putting names? would be the equivalent of (assuming names is null) null == true.  I've reverted the change back.  I think I was future proofing some null argument checks.

**Review by martin-g — APPROVED** [review]

**Comment by martin-g** [comment]
Thank you, @KyleSchoonover !

**Pull request merged**